scripts/solr_updater.py

```diff
@@ -45,7 +45,7 @@ def get_default_offset():
 
 
 class InfobaseLog:
-    def __init__(self, hostname, exclude=None):
+    def __init__(self, hostname: str, exclude: Optional[str] = None):
```

Comment:
Why this change?

Reply:
This was an older change, since undone, that I may have inadvertently re-pushed.